src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,23 @@
+package racingcar;
+
+public class Car {
+    private final String name;
+    private int position;
+
+    public Car(final String name) {
+        if (name.length() > 5) {
+            throw new IllegalArgumentException();
+        }
+        this.name = name;
+    }
+
+    public int getPosition() {
+        return position;
+    }
+
+    public void move(final MovingStrategy movingStrategy) {
+        if (movingStrategy.movable()) {
+            position++;
+        }
+    }
+}
```

Comment:
```suggestion
    }
}

```

https://jojoldu.tistory.com/673 참고해 보시면 좋을 것 같습니다!

Reply:
넵 해당 내용 참고해서 .editorconfig 적용했습니다.
step01 에서 적용한 다음 진행 계속하겠습니다.